integration-cli/docker_cli_service_logs_test.go

```diff
@@ -40,7 +40,7 @@ func (s *DockerSwarmSuite) TestServiceLogs(c *check.C) {
 	// make sure task has been deployed.
 	waitAndAssert(c, defaultReconciliationTimeout,
 		d.CheckRunningTaskImages, checker.DeepEquals,
-		map[string]int{"busybox": len(services)})
+		map[string]int{"busybox:latest": len(services)})
```

Comment:
@dnephin why these changes? I wonder if you take 17.06.1-ce then you may not need to change this.

Reply:
Why do you think this would have reverted in 17.06.1?

Reply:
@tiborvass as you can see from the below output, this is the new behaviour. There's nothing to fix, and it's any different in new releases:

```
$ docker-17.05 service create --name test1705  --label tag-test alpine
tet0jh8j5ygy7me632d3ddrvx

$ docker-17.06 service create --name test1706  --label tag-test alpine
s4cvxf4za8q74nfx5taobs9yk

$ docker-17.06.1 service create --name test17061  --label tag-test alpine
oyyc1i71kn7dusk012frx6ouz

$ docker-17.07 service create --name test1707  --label tag-test alpine
9d0qo4whxcr3fojdk4xwpsd9i

$ docker service ls -f label=tag-test
ID                  NAME                MODE                REPLICAS            IMAGE               PORTS
9d0qo4whxcr3        test1707            replicated          0/1                 alpine:latest
oyyc1i71kn7d        test17061           replicated          0/1                 alpine:latest
s4cvxf4za8q7        test1706            replicated          0/1                 alpine:latest
tet0jh8j5ygy        test1705            replicated          0/1                 alpine
```


```

Reply:
@dnephin I'm concerned about the tests we run for releases cc @andrewhsu @seemethere will this change impact tests on docker-ce ? If so, we need a good solution for the problem before merging.

Reply:
@tiborvass @dnephin Kicked off a build to test integration tests with this PR: https://jenkins.dockerproject.org/job/docker-integration-tests/94/

Reply:
@tiborvass @seemethere we run the release tests using a release binary, so the behaviour will match the updated test code.

This will actually fix two of the reported problems with that suite. See #34623 `TestServiceLogs` and `TestSwarmNetworkPluginV2` are fixed by this change.